### PR TITLE
Include error message when clang-format is not found

### PR DESCRIPTION
**Pull request by adrianno3259**

### CODE REVIEW

File: esss_fix_format/cli.py [hunk 1]
```diff
@@ -165,7 +165,19 @@ def _process_file(filename, check, format_code):
...
         else:
             mtime = os.path.getmtime(filename)
-            subprocess.check_output('clang-format -i "%s"' % filename, shell=True)
+            
+            # checking if the terminal command is valid
+            # might throw an exception if clang-format is not recognized
+            try:
+                subprocess.check_output('clang-format -i "%s"' % filename, shell=True)
+            except subprocess.CalledProcessError as e:
+                errors.append(": ERROR")
```
Comment (prusse-martin) [inline esss_fix_format/cli.py]: This was a left over? A better error message is added to this list in `errors.append(error_msg)`.

Comment (adrianno3259) [reply esss_fix_format/cli.py]: Yes, I'll remove it :)

**Comment by adrianno3259** [comment]
@nicoddemus ,
what do you mean with run esss_fix_format in the code? Are you saying in the tests? And to test it I'll need to reproduce a scenario where "clang-format" is not found. How should I do that? By deactivating the environment inside the unit test?

**Comment by prusse-martin** [comment]
Could you run fix format over your working copy and commit the changes? This will make the builds pass.
(or remove the spaces in blank lines by hand)

**Comment by adrianno3259** [comment]
:D

**Comment by adrianno3259** [comment]
😄

**Pull request merged**